boards/adafruit/feather_nrf52840/doc/index.rst

```diff
@@ -142,6 +142,10 @@ The Feather nRF52840 ships with the `Adafruit nRF52 Bootloader`_ which
 supports flashing using `UF2`_. This allows easy flashing of new images,
 but does not support debugging the device.
 
+To flash the ``adafruit_feather_nrf52480_express`` using UF2, you must use
+the ``adafruit_feather_nrf52480_express_uf2`` board, which uses an appropriate
+flash layout.
```

Comment:
I don't get it, if this is actually the same board, then this should be a variant, not a new board